numpy/linalg/_linalg.py

```diff
@@ -2797,25 +2797,25 @@ def norm(x, ord=None, axis=None, keepdims=False):
         if row_axis == col_axis:
             raise ValueError('Duplicate axes given.')
         if ord == 2:
-            ret = _multi_svd_norm(x, row_axis, col_axis, amax)
+            ret = _multi_svd_norm(x, row_axis, col_axis, functools.partial(amax, initial=0))
```

Comment:
Constructing a partial could slow down computation a bit. Not sure it matters a lot, as this path is already not the fastest.  Maybe measure just to be sure.